### DISCUSSION TITLE
Autocomplete in postgres worsened after 23.2.x (after the syntax highlight update at least)

### DISCUSSION THREAD

**Discussion by Mizar01** [dbeaver/dbeaver#22707] [Q&A]
I noticed that autocomplete is failing at join table fields, and after navigating from word to word to correct some mistakes or add other fields to select/where/on clauses, it totally loose the autocomplete function.

I rarely saw that before the text highlight update.

Recently I only use Postgres dbs. I don't know if it's affecting other kind of db as well.

**Comment by LonwoLonwo** [comment]
Hello @Mizar01 
Can you please provide footage showing your issue? Thank you.
